packages/trace-viewer/src/ui/browserFrame.tsx

```diff
@@ -16,17 +16,31 @@
 
 import './browserFrame.css';
 import * as React from 'react';
+import { useState } from 'react';
+import { CopyToClipboard } from './copyToClipboard';
 
 export const BrowserFrame: React.FunctionComponent<{
   url?: string,
-}> = ({ url }) => {
+}> = ({ url }): React.ReactElement => {
+  const [isHovered, setIsHovered] = useState(false);
```

Comment:
Instead of `useState()` and listening to mouseenter/mouseleave, perhaps we can just use css `:hover` to make the button visible on hover?